eos/form-factors/parametric-bgl1997.hh

```diff
@@ -113,6 +127,8 @@ namespace eos
 
             static std::string _par_name(const std::string & ff_name);
 
+            static const std::vector<OptionSpecification> _options;
```

Comment:
Why do you make the options private?

Reply:
Because I want to avoid accidentally using them outside the interface of an ``ObservableProvider``, i.e., the interface expected by ``ConcreteObservable``. I am a stout "defaults to private" guy!

Reply:
Ok, this should be extended to the other uses of `OptionSpecification` in this case :)

Reply:
Off the top of my head: in all other cases, we have a non-private container of ``OptionSpecification`` within the Implementation class of the ``ObservableProvider``. That class is private by construction.